platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/roster/RosterRetriever.java

```diff
@@ -49,6 +48,7 @@
 public final class RosterRetriever {
     private RosterRetriever() {}
 
+    private static final String ROSTER_SERVICE = "RosterService";
```

Comment:
The `RosterStateId.NAME` and other constants in that class were introduced for a reason so that both the platform and app code could use them. It's not a good idea to replicate string literals when they need to be used in multiple places.

Reply:
In general I certainly agree; but these `Service`/`Schema` keys can never change, so there is no risk of them getting out of sync via multiple references, and it's jarring to have a different place for these keys for `RosterService` specifically; so I prefer this.

Reply:
This approach still establishes a pattern of it being "okay" to replicate these names, and in the future, a typo in one of these or new replicas can easily break things in a way that may be very difficult to debug. So I don't agree with this justification. But if you insist...